tests/baselines/reference/exportAssignmentOfGenericType1.errors.txt

```diff
@@ -1,17 +0,0 @@
-tests/cases/compiler/exportAssignmentOfGenericType1_0.ts(1,10): error TS2449: Class 'T' used before its declaration.

```

Comment:
@sandersn They would - if they weren't already sitting in the folder and correct, because they never got removed when the `.errors` baselines started being written whenever this behavior was first changed. 🐱